modules/core/src/persistence_base64_encoding.cpp

```diff
@@ -202,6 +202,7 @@ int base64::icvCalcStructSize(const char *dt, int initial_size) {
         switch ( *type )
         {
             case 'u': { elem_max_size = std::max( elem_max_size, sizeof(uchar ) ); break; }
+            case 'b': { elem_max_size = std::max( elem_max_size, sizeof(bool  ) ); break; }
```

Comment:
sizeof(bool) may be different on different platforms. to_binary() converts it to uchar.

Reply:
Done